src/Components/Scenes/Chrome/Navigation/Navigation.js

```diff
@@ -0,0 +1,168 @@
+import GovernanceListItem from '../../SideNavigation/GovernanceListItem/GovernanceListItem-Container';
+import HydrationListItem from '../../SideNavigation/HydrationListItem/HydrationListItem-Container';
+import RequestListItem from '../../SideNavigation/RequestListItem/RequestListItem-Container';
+import KeysListItem from '../../SideNavigation/KeysListItem/KeysListItem';
+import AwsAthenaListItem from '../../SideNavigation/AwsAthenaListItem/AwsAthenaListItem';
+import AvatarListItem from '../../SideNavigation/AvatarListItem/AvatarListItem';
+import ExpiredAuth from '../../../Presentational/ErrorSplashes/ExpiredAuth';
+import BadRequest from '../../../Presentational/ErrorSplashes/BadRequest';
+import { useQuery } from '../../../../Hooks/customHooks';
+
+const navWidth = 250;
+
+const navStyle = makeStyles(theme => ({
+  root: {
+    position: 'relative',
+    display: 'flex',
+    flexWrap: 'nowrap',
+    overflow: 'hidden',
+    height: '100vh'
+  },
+  drawer: {
+    width: navWidth,
+    flexShrink: 0,
+    whiteSpace: 'nowrap',
+    position: 'relative',
+    overflow: 'hidden'
+  },
+  drawerOpen: {
+    overflowY: 'auto',
+    overflowX: 'hidden',
+    color: color.g400,
+    backgroundColor: color.black,
+    borderRight: 'none',
+    width: navWidth,
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen
+    }),
+    transitionDelay: '500ms'
+  },
+  drawerClose: {
+    overflow: 'hidden',
+    color: color.g400,
+    backgroundColor: color.black,
+    borderRight: 'none',
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen
+    }),
+    overflowX: 'hidden',
+    width: theme.spacing(7),
+    [theme.breakpoints.up('sm')]: {
+      width: theme.spacing(7)
+    }
+  },
+  customList: {
+    marginBottom: 72
+  }
+}));
+
+const Navigation = props => {
+  const { sessionToken, authenticateFetch } = props;
+  const classes = navStyle();
+
+  const [apiError, setApiError] = useState(null);
+  useEffect(() => {
+    axios.interceptors.response.use(
+      response => {
+        console.log('good response');
+        setApiError(null);
+        return response;
+      },
+      error => {
+        console.log('bad response');
+        setApiError(error.response.status);
+        return Promise.reject(error);
+      }
+    );
+  });
+
+  // Case 1: there is a SAML response but no session token, so authenticate real quick
+  // Case 2: there is neither a SAML response nor a session token, so redirect to OneLogin
+  // Case 3: there may or may not be a SAML response,
+  //         but there is a session token, so relax until an hour later
+  //         when we get a 4xx code from some request, then redirect
+  const samlResponse = useQuery('SAMLResponse');
+  const [isRedirecting, setIsRedirecting] = useState(true);
```

Comment:
nitpick: just the naming isRedirecting implies this should be true when redirecting to OneLogin. Maybe something to indicate that the API returned an error code like `errorPage`

Reply:
Let's sync up on this but I believe it is set to true when it does redirect - the case here is to detect the "first" time a user lands, not to auto-redirect on something like an `errorPage` encountered. That view will have a manual link out to OneLogin if Scott's designs stay relatively the same :)

Reply:
Per our convo Friday I am good with the naming here